app/models/drupal_tag.rb

```diff
@@ -170,4 +170,9 @@ def self.find_research_notes(tagnames, limit = 10)
               .limit(limit)
   end
 
+  # Returns author of tag
+  def author
+    self.drupal_node_community_tag.first.author
```

Comment:
Actually, now that I think about this again -- i was wrong! This would return the author of the first person to ever have used a tag of this name. I think, unfortunately, we need to remove these four lines, and solve it another way. I'm so sorry!